tests/integration/popular_repos/BUILD.bazel

```diff
@@ -173,7 +172,6 @@ test_suite(
         "@org_golang_x_tools//go/callgraph:callgraph_test",
         "@org_golang_x_tools//go/callgraph/static:static_test",
         "@org_golang_x_tools//go/callgraph/vta/internal/trie:trie_test",
-        "@org_golang_x_tools//go/cfg:cfg_test",
```

Comment:
@fmeum This test was the only one that was removed because it was failing (the others were removed).

This one failed with:
```
==================== Test output for @org_golang_x_tools//go/cfg:cfg_test:
--- FAIL: TestSmoke (0.01s)
    cfg_test.go:180: go tool not available: 'go env GOROOT' does not match runtime.GOROOT:
                go env: /home/user/.cache/bazel/_bazel_tfrench/eb80ee7746c8e42f65d76b9b55d4ad41/external/go_sdk
                GOROOT: GOROOT
FAIL
================================================================================
Target @org_golang_x_tools//go/cfg:cfg_test up-to-date:
```
Is this expected behavior

Reply:
Yes: We build the binaries with a fake GOROOT as they could otherwise contain absolute paths. If we could pass in the relative path to the SDK instead, that may be better